build/config/src/main/resources/checkstyle.xml

```diff
@@ -13,12 +13,12 @@
     <property name="cacheFile" value="${checkstyle.cache.file}" />
 
     <module name="Header">
-        <property name="header" value="/*\n * Hibernate Search, full-text search for your domain model\n *\n * License: GNU Lesser General Public License (LGPL), version 2.1 or later\n * See the lgpl.txt file in the root directory or &lt;http://www.gnu.org/licenses/lgpl-2.1.html&gt;.\n */"/>
+        <property name="headerFile" value="classpath:/org/hibernate/search/headers/java.header"/>
         <property name="fileExtensions" value="java"/>
```

Comment:
Would that include Jenkins files as well?

also on a related note, do we need to "license" docker files too?

Reply:
Nope, I forgot about checking those -- I'll add something.
I hope your Dockerfile parsing utilities take comments into account -- let's see ;D

Reply:
Actually neither Jenkinsfiles nor Dockerfiles are part of a Maven project with a checkstyle execution... They're technically part of the root project (I guess), and I'm not sure what the effect of adding a checkstyle to the root project would be, so... I'll leave this out for now.